script/sync_contributors.rb

```diff
@@ -9,60 +9,57 @@ def fetch_contributors
   all_contributors = []
   page = 1
   per_page = 100 # Maximum allowed by GitHub API
-  
+
   loop do
     uri = URI("https://api.github.com/repos/viewcomponent/view_component/contributors")
     uri.query = URI.encode_www_form(page: page, per_page: per_page)
-    
+
     contributors = Net::HTTP.start(uri.host, uri.port, use_ssl: true) do |http|
       request = Net::HTTP::Get.new(uri)
       # Add User-Agent header as GitHub API recommends it
       request["User-Agent"] = "ViewComponent Contributors Sync Script"
-      
+
       response = http.request(request)
-      
+
       if response.code == "200"
         JSON.parse(response.body)
       else
         puts "Error fetching contributors (page #{page}): #{response.code} #{response.message}"
         exit 1
       end
     end
-    
+
     # Break if no more contributors on this page
     break if contributors.empty?
-    
+
     all_contributors.concat(contributors)
     puts "Fetched page #{page} with #{contributors.length} contributors"
-    
+
     # Break if we got less than the full page size (last page)
     break if contributors.length < per_page
-    
+
     page += 1
   end
-  
+
   all_contributors
 end
 
 # Transform contributors data for YAML output
 def transform_contributors(contributors)
   usernames = contributors.map do |contributor|
-    {
-      "login" => contributor["login"],
-      "html_url" => contributor["html_url"]
-    }
-  end
-  
+    contributor["login"]
+  end.sort
```

Comment:
Consider sorting case-insensitively unless it was intentional to have all capital-first-letter usernames appear at the beginning.